### PR TITLE
Adapted e2e tests and fixed some integrations

### PR DESCRIPTION
**Pull request by MatiBrego**

### CODE REVIEW

File: src/services/snippet.service..ts [hunk 1]
```diff
@@ -92,7 +92,7 @@ export class SnippetService implements SnippetOperations {
     }
 
     shareSnippet(snippetId: string, userId: string): Promise<Snippet> {
-        return axiosInstance.post(`${MANAGER_URL}/share`, adaptShareSnippet(snippetId, userId))
+        return axiosInstance.post(`${MANAGER_URL}/manager/share`, adaptShareSnippet(snippetId, userId))
```
Comment (lucaslovaglio) [inline src/services/snippet.service..ts]: In this case '/manager' is included in the url

**Review by lucaslovaglio — CHANGES_REQUESTED** [review]
in the snippet.service the '/manager' is included in the manager url. 
The other files LGTM

**Comment by MatiBrego** [comment]
> in the snippet.service the '/manager' is included in the manager url. 

I think it should not be included in the url, since there are some paths that do not include it, such us '/case' or '/rule'.

**Review by lucaslovaglio — APPROVED** [review]
LGTM

**Pull request merged**